server/statistics/hot_peer.go

```diff
@@ -26,9 +26,15 @@ import (
 const (
 	ByteDim int = iota
 	KeyDim
+	QueryDim
 	DimLen
 )
 
+func IsSelectedDim(dim int) bool {
```

Comment:
What is the purpose of this function?

Reply:
This function means if this dimension is used in hotspot scheduling, it will be used when determining hotspots and calculating expectations. This pr will only introduce the statistics of qps and will not address the role of qps in scheduling.

Reply:
The name of this function and its implementation don't match.

Reply:
Do you have any good ideas about the name of this function?